src/cljs/main/broadfcui/page/workspace/notebooks/create_cluster.cljs

```diff
@@ -0,0 +1,208 @@
+(ns broadfcui.page.workspace.notebooks.create_cluster
+  (:require
+   [dmohs.react :as react]
+   [broadfcui.utils :as utils]
+   [broadfcui.common.icons :as icons]
+   [broadfcui.common.flex-utils :as flex]
+   [broadfcui.components.buttons :as buttons]
+   [broadfcui.common.components :as comps]
+   [broadfcui.endpoints :as endpoints]
+   [broadfcui.utils.ajax :as ajax]
+   [broadfcui.components.modals :as modals]
+   [broadfcui.components.blocker :refer [blocker]]
+   [broadfcui.common.links :as links]
+   [broadfcui.common.style :as style]
+   [broadfcui.components.collapse :refer [Collapse]]
+   [broadfcui.common.input :as input]
+   [broadfcui.components.foundation-tooltip :refer [FoundationTooltip]]
+   [broadfcui.page.workspace.notebooks.utils :as notebook-utils]
+   [clojure.string :as string]
+   ))
+
+(def machineTypes ["n1-standard-1"
+                   "n1-standard-2"
+                   "n1-standard-4"
+                   "n1-standard-8"
+                   "n1-standard-16"
+                   "n1-standard-32"
+                   "n1-standard-64"
+                   "n1-standard-96"
+                   "n1-highcpu-4"
+                   "n1-highcpu-8"
+                   "n1-highcpu-16"
+                   "n1-highcpu-32"
+                   "n1-highcpu-64"
+                   "n1-highcpu-96"
+                   "n1-highmem-2"
+                   "n1-highmem-4"
+                   "n1-highmem-8"
+                   "n1-highmem-16"
+                   "n1-highmem-32"
+                   "n1-highmem-64"
+                   "n1-highmem-96"])
+
+(react/defc ClusterCreator
```

Comment:
This code is essentially copied from here: https://github.com/broadinstitute/firecloud-ui/blob/develop/src/cljs/main/broadfcui/page/workspace/notebooks/notebooks.cljs#L82